components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
+  return (
+    <StyledButton
+      onClick={() => {
+        setProduct(produkt);
+        console.log(productToBuy);
+      }}
+    >
+      Buy
+    </StyledButton>
```

Comment:
Better: clean up console.logs. Be consistent with naming conventions.

```suggestion
    <StyledButton
      onClick={() => {
        setProduct(product);
      }}
    >
      Buy
    </StyledButton>
```